src/components/questionStructure/Task/variants/HandwritingTask/HandwritingEditor.tsx

```diff
@@ -5,27 +5,33 @@ import React from 'react'
 import Markdown from '../../../../Markdown'
 import Canvas from './Canvas'
 import './handwritingEditor.css'
+import { HandwritingAnswer } from './types'
 
 interface HandwritingEditorProps {
   username: string
-  latex: string
+  answer?: HandwritingAnswer
   onAnswerChange: (value: string) => void
 }
 
 const HandwritingEditor: React.FC<HandwritingEditorProps> = ({
   username,
-  latex,
+  answer,
   onAnswerChange,
 }) => {
   return (
     <Flex direction="column" flexGrow="1" align="center" gap="3">
       <Card className="mathjax-card">
         <Box p="3">
-          <MathJax>{`\\( ${latex} \\)`}</MathJax>
+          <MathJax>{`\\( ${answer?.latex} \\)`}</MathJax>
```

Comment:
Do we need to worry about default values? 
```suggestion
          <MathJax>{`\\( ${answer?.latex ?? ''} \\)`}</MathJax>
```